hr_attendance_modification_tracking/models/hr_attendance.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 Creu Blanca
+# Copyright 2021 Landoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+
+class HrAttendance(models.Model):
+    _name = "hr.attendance"
+    _inherit = ["hr.attendance", "mail.thread"]
+
+    employee_id = fields.Many2one(tracking=True)
+    check_in = fields.Datetime(tracking=True)
+    check_out = fields.Datetime(tracking=True)
+    time_changed_manually = fields.Boolean(
+        string="Time changed",
+        default=False,
+        readonly=True,
+        help="This attendance has been manually changed by user. If attendance"
+        " is created from form view, a 60 seconds tolerance will "
+        "be applied.",
+    )
+
+    @api.model_create_multi
+    def create(self, vals):
+        tolerance = timedelta(seconds=60)
+        now = fields.Datetime.now()
+        for val in vals:
+            for check in ["check_in", "check_out"]:
+                if (
+                    val.get(check, False)
+                    and abs(fields.Datetime.from_string(val.get(check)) - now)
+                    > tolerance
+                ):
+                    val.update({"time_changed_manually": True})
+                    break
```

Comment:
I'd call the initial `vals` --> `vals_list` and then what you called `val` is actually `vals` or `vals_dict`